suite-native/thp/src/hooks/useThpAutoconnectActions.tsx

```diff
@@ -0,0 +1,28 @@
+import { useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { startThpAutoconnectThunk, thpActions } from '@suite-common/thp';
+import { selectSelectedDevice } from '@suite-common/wallet-core';
+
+export const useThpAutoconnectActions = () => {
+    const dispatch = useDispatch();
+
+    const device = useSelector(selectSelectedDevice);
+
+    const startAutoconnect = useCallback(async () => {
```

Comment:
Might add `Thp` to the name:

```suggestion
    const startThpAutoconnect = useCallback(async () => {
```

Reply:
Fixed